File: Aplicacao_cliente/SilentManager/app/src/main/java/com/silent_manager/g29/silent_manager/data_layer/models/Event.kt
package com.silent_manager.g29.silent_manager.data_layer.models

import android.arch.persistence.room.Entity
import android.os.Parcel
import android.os.Parcelable
import com.fasterxml.jackson.annotation.JsonProperty
import java.io.Serializable
import java.util.*

@Entity(tableName = "Events", primaryKeys = arrayOf("eventId"))
data class Event(
    @JsonProperty("EventId")
    val eventId: Int?,
    @JsonProperty("Name")
    val name: String?,
    @JsonProperty("Description")
    val description: String?,
    @JsonProperty("StartDate")
    val startDate: Date?,
    @JsonProperty("EndDate")
    val endDate: Date?,
    @JsonProperty("Author")
    val author: User?,
    @JsonProperty("Location")
    val location: Location?,
    @JsonProperty("Radius")
    val radius: Int?,
    @JsonProperty("State")
    val state: Int?,
    @JsonProperty("Category")
    val category: String?
) : Serializable

File: Servidor/SilentManagerApiServer/Models/DB/Event.cs
using System;
using System.Collections.Generic;

namespace SilentManagerApiServer.Models.DB
{
    public partial class Event
    {
        public Event()
        {
            EventHasCategories = new HashSet<EventHasCategories>();
            Invite = new HashSet<Invite>();
        }

        public int EventId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? EndDate { get; set; }
        public int? Author { get; set; }
        public int? Location { get; set; }
        public int? State { get; set; }
        public int? Radius { get; set; }

        public virtual User AuthorNavigation { get; set; }
        public virtual Location LocationNavigation { get; set; }
        public virtual StateEvent StateNavigation { get; set; }
        public virtual ICollection<EventHasCategories> EventHasCategories { get; set; }
        public virtual ICollection<Invite> Invite { get; set; }
    }
}
